README.rst

```diff
@@ -65,6 +65,13 @@ Features
 
 -----
 
+=============
+Prerequisites
+=============
+
+* Python >= 3.4
+* pip >= 8.0
+
```

Comment:
after this newline, add a ``-----`` so that a horizontal line appears so it splits the section :D